### ISSUE TITLE
API request was aborted

### ISSUE THREAD

**Issue by csqCory** [MetacoSA/QBitNinja#63]
'QBitNinja.Listener.Console --init' The operation is not completed, can you request API

**Comment by NicolasDorier** [comment]
I advise you to not use QBitNinja for new development, take a look at https://github.com/dgarage/NBXplorer . It is easier to run, and lightweight.

**Comment by csqCory** [comment]
OK, but I spent a week in deploying QBitNinja. I want to see the final result of it. I hope you can help me.

**Comment by NicolasDorier** [comment]
Yes, that's the downside of using qbitninja, it is very long to setup. Have you correctly configured your configuration file?

**Comment by csqCory** [comment]
![img](https://user-images.githubusercontent.com/40781614/42375057-ee0b8f5a-814c-11e8-8467-79b1f43c900a.png)
I configure it like this, look at it

**Comment by NicolasDorier** [comment]
can you copy paste here all the logs you have
